config/application.yml.example

```diff
@@ -0,0 +1,22 @@
+# Add configuration values here, as shown below.
+#
+# pusher_app_id: "2954"
+# pusher_key: 7381a978f7dd7f9a1117
+# pusher_secret: abdc3b896a0ffb85d373
+# stripe_api_key: sk_test_2J0l093xOyW72XUYJHE4Dv2r
+# stripe_publishable_key: pk_test_ro9jV5SNwGb1yYlQfzG17LHK
+#
+# production:
+#   stripe_api_key: sk_live_EeHnL644i6zo4Iyq4v1KdV9H
+#   stripe_publishable_key: pk_live_9lcthxpSIHbGwmdO941O1XVU
+
+# See Figaro documentation for 
+
+SMTP_USERNAME: 'patmbolger'  # username for sendgrid account 
+SMTP_PASSWORD: '38LjEPStn-'  # password for sendgrid account
+SMTP_DOMAIN:  'herokuapp.com'
+SMTP_ADDRESS: 'smtp.sendgrid.net'
+
+order_receipt_from_email: 'tma_kitchen@gmail.org'
+
+
```

Comment:
Christian - yes, since we use the 'order_receipt_from_email' ENV var as a fallback in case the client has not configured (in AA) one or both of 1) default "from" address for emails to customer, and 2) one or more addresses to receive email notification that a customer order has been received.